src/connector/src/source/base.rs

```diff
@@ -138,27 +138,29 @@ pub struct SourceCtrlOpts {
     pub rate_limit: Option<u32>,
 }
 
-impl Default for SourceCtrlOpts {
```

Comment:
May `impl !Default for SourceCtrlOpts {}` and document the reason.

Reply:
Good idea! Fixed.